### ISSUE TITLE
Using Crtl + Alt in a selection to copy and snap a selection to the grid

### ISSUE THREAD

**Issue by SorceryStory** [aseprite/aseprite#778]
Right now, when you have a part of the canvas selected, you can copy that selection by Ctrl+dragging. Also, you can move the selection in the grid by dragging while pressing Alt. But when you press both at the same time, it defaults to a simple drag functionality.

You can get this by simply dragging with the mouse (unless I'm missing something), so what I'd like to suggest is to combine both functionalities when pressing both Ctrl and Alt, effectively copying the selection and dragging it in the grid.

You can already do this if you press Alt after starting to drag while holding Ctrl. However, you need to let go of Ctrl and then press Alt in order for the grid drag to take effect. It would be a lot smoother if you could do both things immediately when pressing both keys.

So, to summarize:

What we need to do right now in order to copy and place a selection in the grid:
Press Ctrl, Drag to desired location, Let go of Ctrl, Press Alt, position where wanted within the grid;

And I'm proposing the following:
Press both Alt and Ctrl, drag to desired location and drop within the grid.

I hope that made sense! Here's a small image to illustrate what I mean:
![2015-08-22 20 32 55_aseprite_func](https://cloud.githubusercontent.com/assets/13921479/9425720/48a60bd6-4913-11e5-9bf7-6f4a6d3f316a.png)

(Also, I'm using windows here to demonstrate. Other OSs would use their own specific bindings.)
